diff -ruNp acpid-2.0.10.orig/input_layer.c acpid-2.0.10/input_layer.c
--- acpid-2.0.10.orig/input_layer.c	2011-03-12 17:22:34.000000000 +0100
+++ acpid-2.0.10/input_layer.c	2014-10-23 21:15:00.054570789 +0200
@@ -115,6 +115,10 @@ static struct evtab_entry evtab[] = {
  		"button/volumeup VOLUP 00000080 00000000"},
 	{{{0,0}, EV_KEY, KEY_MUTE, 1}, 
  		"button/mute MUTE 00000080 00000000"},
+	{{{0,0}, EV_KEY, KEY_MICMUTE, 1}, 
+ 		"button/micmute MICMUTE 00000080 00000000"},
+	{{{0,0}, EV_KEY, KEY_F20, 1}, 
+ 		"ibm/hotkey HKEY 00000080 0000101b"},
  	/* cd play/pause buttons */
  	{{{0,0}, EV_KEY, KEY_NEXTSONG, 1}, 
  		"cd/next CDNEXT 00000080 00000000"},
